css-canvas-medical/index.html

```diff
@@ -0,0 +1,127 @@
+      <nav>
+        <ul>
+          <li>
+            <a href="product.html">Product</a>
+          </li>
+          <li>
+            <a href="implementation.html">Implementation</a>
+          </li>
+          <li>
+            <a href="#">About Us</a>
+          </li>
+          <li>
+            <a href="#">Blog</a>
+          </li>
+          <li><button id="navbtn">Contact Us</button></li>
+        </ul>
+      </nav>
+    </div>
+  </div>
+  <hr />
+  <div class="container">
+    <div class="backgroundimg">
+      <div class="col-six-over-ten add-padding-80">
+        <h1 class="adjust-font-color-weight">The EMR Reimagined for Primary Care</h1>
+        <p class="adjust-font-color-size-family">Can't remember the last time you talked with a patient without focusing
+            on the EMR? Canvas lets you rediscover joy in the work you do. Capture meaningful data,
+            identify gaps in care, and move seamlessly between tasks within the platform. You stay present with
+            patients while your whole care team stays up-to-speed. Be a better doctor with Canvas, designed
+            for primary care providers with the help of primary care providers.
+        </p>
+        <button class="btn">Get Started</button>
+      </div>
+    </div>
+  </div>
+  <div class="backgroundcolor">
+    <div class="row justify-content-spacebetween">
+      <div class="align-content-center col-two-fifth">
+        <h2 class="adjust-heading-font-color-size-family-weight">Clinicians Prefer Canvas</h2>
+        <p class="adjust-subheading-color-line-height">Physician-owned primary care practices across the country have already made Canvas the fabric of their care.</p>
+      </div>
+    </div>
+    <div class="container">
+      <div class="row justify-content-spacebetween">
+        <div class="col-one-half add-font-family">
+          <h3 class="adjust-font-weight-size-color-position">Cascade Family Practice, Portland OR</h3>
+          <h3 class="adjust-font-weight-size-color-position">Temescal Creek Medicine, Oakland CA</h3>
+          <h3 class="adjust-font-weight-size-color-position">San Juan Healthcare, Friday Harbor</h3>
+        </div>
+        <div class="col-one-half add-font-family">
+          <h3 class="adjust-font-weight-size-color-position">Ward Medical Clinic, Bonifay FL</h3>
+          <h3 class="adjust-font-weight-size-color-position">Caribou Medical, Towanda PA</h3>
+          <h3 class="adjust-font-weight-size-color-position">Bay Tree Family Care, Red Bay AL</h3>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="container">
+    <div class="row learn-more justify-content-spacebetween">
+      <div class="align-content-left">
+        <h3>Ready to learn more?</h3>
+        <p>Schedule a private meeting now</p>
+      </div>
+      <div class="align-content-right">
+        <button class="btn-right">Get Started</button>
+      </div>
+    </div>
+  </div>
+  <div class="footer">
+    <div class="container row">
```

Comment:
No one element should be both a `container` and a `row`, I believe I mentioned this in a previous review.